conf/config.json.example

```diff
@@ -7,6 +7,7 @@
         "loglevel": "info",
         "useCDN": false,
         "allowGravatar": false,
+        "allowPDFExport": true,
```

Comment:
For security reasons, the PDF export has been removed in upstream  version since v.1.5. This feature may be back on v. 2.0

Reply:
Could you link to the explanation of that issue ? I'm interested to know.

Reply:
It is mentioned here https://demo.codimd.org/s/release-notes# (see version 1.5) and its was mentioned on their Rio channel. Not much explanation though...

Reply:
Ok, then let's remove it :)